src/components/SharedStyledComponents.js

```diff
@@ -443,3 +445,28 @@ export const dropdownIconContainerVariant = {
   },
   closed: { rotate: -90, y: 0 },
 }
+
+// Common styled item card for languages/about
+
+export const CardItem = styled(GatsbyLink)`
+  text-decoration: none;
+  margin: 1rem 1rem 1rem 0;
+  padding: 1rem;
+  flex: 1 1 200px;
```

Comment:
I don't think this quite works.

It looks great for the in-progress translation cards:
https://share.getcloudapp.com/yAu6QOkE

But it warps the styles of the language cards (see your preview deploy: https://deploy-preview-2481--ethereumorg.netlify.app/en/languages/): 
https://share.getcloudapp.com/bLug5EBE
Those cards likely need that default `width: 240px;` property. Perhaps you overwrite these base styles in that file (e.g. `LangItem = styled(CardItem)`...

Reply:
Got it! I'll overwrite it for the language card. Also do let me know if you want to get it changed for the work in progress cards on `/about`.